joomla/joomla-cms
# [4.0] Template Atum - Change Alerts styling - Space saving

### Summary of Changes
Changed the Header of the message to be next to the message to save space
Decreased paddings
Changed colors

### Testing Instructions
See if you like the new style and if not, tell me what you would prefer instead.

Before:
![grafik](https://user-images.githubusercontent.com/828371/81217622-30f30a00-8fdd-11ea-8e07-1da12aaf714b.png)

After:
![grafik](https://user-images.githubusercontent.com/828371/81220497-ac56ba80-8fe1-11ea-86f9-ffb30ef8d938.png)
![grafik](https://user-images.githubusercontent.com/828371/81220519-b7a9e600-8fe1-11ea-9c66-6ad4121eefeb.png)

thank you @dgrammatiko for the hints where to change everything :-)
